components/imageInput.tsx

```diff
@@ -0,0 +1,27 @@
+import type React from 'react'
+import { type ComponentPropsWithRef, forwardRef } from 'react'
+import type { FC, InputHTMLAttributes } from 'react'
+
+export type Props = ComponentPropsWithRef<'input'> & {
```

Comment:
This export property is not used anywhere now.
I think `export` should be removed.

Reply:
I removed `export`.